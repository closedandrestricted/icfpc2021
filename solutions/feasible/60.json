{"vertices":[[43,31],[27,46],[20,16],[29,16],[6,22],[23,19],[17,10],[9,17],[9,25],[19,20],[5,4],[9,13],[11,7],[4,25],[6,11],[5,7],[6,2],[10,26],[15,33],[2,2],[8,7],[11,11],[20,20],[9,4],[7,14],[13,18],[29,31],[10,14],[11,15],[20,13],[24,24],[23,24],[11,19],[16,33],[51,23],[7,14]]}