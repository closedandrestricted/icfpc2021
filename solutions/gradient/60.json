{"vertices":[[4,69],[7,14],[10,38],[24,60],[40,1],[23,37],[51,12],[21,38],[54,30],[47,21],[35,39],[41,47],[44,65],[57,81],[42,37],[39,54],[40,59],[58,20],[49,69],[51,33],[53,55],[50,59],[54,10],[56,38],[55,48],[43,57],[36,76],[58,20],[59,1],[70,38],[50,14],[57,77],[81,35],[66,66],[78,95],[88,65]]}