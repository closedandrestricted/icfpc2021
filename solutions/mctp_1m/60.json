{"vertices":[[47,49],[10,10],[43,39],[59,67],[33,36],[24,33],[49,40],[34,45],[62,50],[51,50],[47,41],[47,51],[38,45],[58,45],[54,39],[43,45],[43,40],[54,61],[60,31],[49,48],[48,41],[53,42],[64,66],[44,53],[41,43],[58,47],[74,35],[60,56],[72,72],[31,47],[59,82],[51,18],[20,44],[21,36],[30,34],[27,56]]}