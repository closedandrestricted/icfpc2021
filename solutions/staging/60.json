{"vertices":[[54,8],[41,3],[43,68],[45,28],[12,13],[55,27],[25,23],[49,59],[42,36],[34,28],[37,52],[31,44],[40,50],[24,63],[44,50],[36,39],[39,43],[42,36],[11,67],[54,49],[47,55],[46,50],[49,45],[59,44],[54,53],[44,57],[24,61],[55,35],[53,36],[73,42],[47,22],[30,40],[84,39],[61,73],[3,38],[81,82]]}