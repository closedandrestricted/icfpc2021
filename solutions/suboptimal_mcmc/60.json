{"vertices":[[49,1],[86,39],[6,9],[58,20],[1,44],[60,38],[12,32],[16,15],[30,42],[22,34],[29,22],[38,27],[46,21],[66,25],[33,28],[44,23],[39,23],[32,40],[80,27],[37,37],[44,34],[39,33],[43,41],[30,38],[40,40],[46,35],[66,29],[38,51],[53,39],[31,52],[69,43],[40,57],[27,62],[23,53],[63,73],[6,67]]}