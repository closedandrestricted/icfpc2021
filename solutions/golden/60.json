{"vertices":[[50,1],[64,73],[21,64],[48,23],[39,63],[41,57],[39,47],[22,53],[33,40],[30,51],[26,40],[30,31],[25,31],[9,44],[19,41],[36,35],[31,35],[28,40],[5,31],[20,31],[24,40],[27,44],[24,30],[19,38],[28,34],[20,43],[1,45],[33,36],[32,24],[15,52],[47,21],[47,42],[9,62],[11,24],[75,41],[1,5]]}